Source/AviationWeather/Inc/AviationWeather/metar.h

```diff
@@ -303,6 +303,7 @@ class metar_info
     cloud_layer ceiling() const;
 
     flight_category flight_category() const;
+    int8_t temperature_dewpoint_spread() const;
```

Comment:
Because `temperature` and `dewpoint` are `int8_t`, `temperature - dewpoint` could become too large for the `int8_t` representing `temperature_dewpoint_spread`.  Dew point is *should* always be less than or equal to temperature, so `uint8_t` could work, but that doesn't account for invalid data.